packages/blade/src/components/Carousel/_decisions/decisions.md

```diff
@@ -0,0 +1,150 @@
+| navigationButtonStyle    | `filled,stroke`                        | `filled`                                 | Styles to be used depending on the emphasis you want to give to the navigation buttons                                                                                                       |          |
+| indicatorVariant         | `gray,white,blue`                      | `gray`                                   | Variants to be used depending on the background and the emphasis you want to give to the indicator.                                                                                          |          |
+| onChange                 | `(slideIndex: number) => void`         | `undefined`                              | Callback which gets fired everytime a slide changes, useful for analytic events (only runs on user interaction not on autoPlay)                                                              |          |
+
+### CarouselItem
+
+- N/A - CarouselItem will just be a wrapper and have internal styling
+
+## Usage Example
+
+```jsx
+const Testimonial = () => {
+  return (
+    <Box display="flex" gap="spacing.2">
+      <Box>
+        <Title>Acquire Customers From New Customer Segments</Title>
+        <Text weight="bold">Subham Kumar</Text>
+      </Box>
+      <Img src="./avatar-1.png" />
+    </Box>
+  )
+};
+
+<Carousel
+  autoPlay
+  visibleItems={2}
+  navigationButtonPosition="side"
+  bleed="right"
+>
+  <CarouselItem>
+    <Testimonial />
+  </CarouselItem>
+
+  <CarouselItem>
+    <Testimonial />
+  </CarouselItem>
+
+  <CarouselItem>
+    <Testimonial />
+  </CarouselItem>
+</Carousel>
+```
+
+<img src="./example-usage-1.png" width="70%" />
+
+### Usage with Card
+
+
+```jsx
+const TestimonialCard = () => {
+  return (
+   <Card elevation="highRaised" padding="spacing.0">
+      <CardBody>
+        <Box display="flex" flexDirection="row">
+          <img
+            width="300"
+            height="auto"
+          />
+          <Box padding="spacing.7" display="flex" flexDirection="column">
+            <Heading size="large">Subham Kumar</Heading>
+            <Text marginTop="spacing.5">
+            Acquire Customers From New Customer Segments
+            </Text>
+            <Link href="#">Learn More</Link>
+          </Box>
+        </Box>
+      </CardBody>
+    </Card>
+  )
+};
+
+<Carousel
+  autoPlay
+  visibleItems={2}
+  navigationButtonPosition="side"
+  bleed="right"
+>
+  <CarouselItem>
+    <TestimonialCard />
+  </CarouselItem>
+
+  <CarouselItem>
+    <TestimonialCard />
+  </CarouselItem>
+
+  <CarouselItem>
+    <TestimonialCard />
+  </CarouselItem>
+</Carousel>
+```
+
+## Accessibility
+
+We need to follow the APG Pattern For [Carousel](https://www.w3.org/WAI/ARIA/apg/patterns/carousel/examples/carousel-2-tablist/)
+
+## Motion
```

Comment:
Does the navigation button go in a disabled state when reached either ends or does it keep looping?

Reply:
From the motion docs, it seems that it loops - but i'll confirm the UX once with RK (ideally it should loop tho)

Reply:
In normal variants it loops, but in case of `visibleItems: autofit & shouldAddStartEndSpacing` we remove the next/prev button.